src/Button/index.js

```diff
@@ -1 +1,2 @@
 export { default } from './Button';
+export { Variant as ButtonVariant } from './variants';
```

Comment:
There are many more types we'll want to export eventually, but I made it a point to export this as it was already called out as necessary in the main app.